src/audio_generic.cpp

```diff
@@ -182,11 +194,46 @@ bool GenericAudio::PlayOnChannel(BgmChannel& chan, Filesystem_Stream::InputStrea
 		return false;
 	}
 
+	// Midiout is only supported on channel 0 because this is an exclusive resource
+	if (chan.id == 0 && GenericAudioMidiOut::IsSupported(filestream)) {
+		chan.decoder.reset();
+
+		// FIXME: Try Fluidsynth and WildMidi first
```

Comment:
Is this fixme related to how to reorder priority when games provide soundfonts/patches?

Reply:
Yes, right now it is Fluidsynth, Wildmidi, Native, FmMidi.

FmMidi is kinda unused when Native is available. There is a configuration option needed.